src/main.cpp

```diff
@@ -33,8 +18,16 @@ Server& initServer()
 int main(int argc, char **argv)
 {
 	if (argc != 2 || !argv[1])
-	{
-		return 0;
+	{	
+		try
+		{
+			std::cout << "This is a placeholder for the default config.";
+			return 0;
+		}
+		catch(const std::logic_error& e)
+		{
+			std::cerr << e.what() << '\n';
+		}
```

Comment:
I dont really get whats going on here, are we not going to add this parsing into the same try catch as the rest? this also now just exits straight up if we dont provide an argument as input. Could just be taken out and just cerr "Expecting at least one input" or smt..